modules/installation-gcp-permissions.adoc

```diff
@@ -32,6 +32,13 @@ account requires the following permissions. If you deploy your cluster into an e
 .Required roles for creating network resources during installation
```

Comment:
If either `platform.gcp.publicDNSZone` or `platform.gcp.privateDNSZone` parameters are not specified (in `install-config.yaml`), DNS record-sets / private zone would be configured in the service project, in this case the google cloud credential would need `DNS Administrator` role in the service project.

Reply:
I added an extra note to callouts 7 and 9 in the sample yaml with this information.